openshift/README.md

```diff
@@ -7,41 +7,49 @@ Steps
 
 1. Unless you have built OpenShift locally, be sure  to grab the [oc command, v1.3+](https://github.com/openshift/origin/releases/tag/v1.3.1)
 
-1. Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
+1. Stand up an OpenShift cluster from origin master, installing the standard image streams to the OpenShift namespace:
 
-        $ oc cluster up
+        oc cluster up
 
 1. Login as a normal user (any non-empty user name and password is fine)
 
-        $ oc login
+        oc login
 
 1. Create a project  named "test"
 
-        $ oc new-project test
+        oc new-project test
 
 1. Create persistent volume 
 
-**Note**: jenkins-persistent-template.json template file requires openshift persistent volume.
+**Note**: jenkins-persistent-template.json template file requires an OpenShift persistent volume.
 Persistent volume setup is not part of the template and require separate steps. 
 If you already have persistent volumes feel free to skip this step.
+
 To setup simple persistent volume on new cluster execute:
         
-        $ mkdir /tmp/jenkins
-        $ chmod 777 /tmp/jenkins
-        $ oc login -u system:admin
-        $ oc create -f ./sample-pv.json
+        mkdir /tmp/jenkins
+        chmod 777 /tmp/jenkins
+        # creating a cluster wide persistent volume like the one we use requires
```

Comment:
Sorry for not being clear. I think that we can just simply say: `Use cluster admin account to create PV.`